volume17.xml

```diff
@@ -0,0 +1,215 @@
+                <dateline>
+                    <date when="1911-03-12">March 13th. Monday.</date>
+                </dateline>
+                <p>Mr. <persName>Draper</persName> to lunch yesterday. At home all day &#8211; rather warm today. <persName>El&#232;na</persName> and I drove into town this morning &#8211; saw a number <pb n="83"/>of friends &#8211; met <persName>Mr. Newberry</persName> at <placeName>Deimers,</placeName> and he went with us to <persName>Kolekians</persName> and to <persName>Kasseras</persName> &#8211; there <persName ref="#Davis_Theodore_M">Theo</persName> and <persName>Harry</persName> joined us, and we went to Nachmans. <persName>El&#232;na</persName> and <persName>Harry</persName> went after lunch to the <placeName>Mena House,</placeName> and <persName ref="#Davis_Theodore_M">Theo</persName> and I went for a drive &#8211; and finding ourselves, in spite of warm wraps, freezing in the cold wind, we came home.</p>
+                <dateline>
+                    <date when="1911-03-14">March 14th. Tuesday.</date>
+                </dateline>
+                <p>At home all day &#8211; still cold and windy. Wednesday. March 15th. <persName>El&#232;na</persName> and <persName>Harry</persName> made and early start for <name>Sakkara</name> &#8211; going by train to the <placeName>Mena House,</placeName> and taking donkeys for <orgName>Sakkara.</orgName> They came back at 7 P.M. after a joyful day. Mr. <persName>Winlock</persName> appeared this morning at 10 &#8211; and made quite a visit &#8211; then he, <persName ref="#Davis_Theodore_M">Theo</persName> and I went to town, and to <placeName>Kyticas's</placeName> to see a lovely <orgName>Greek</orgName> statuette which <persName ref="#Davis_Theodore_M">Theo</persName> thinks of buying. Mr. and <persName>Mrs. Newberry</persName> and Mr. <persName>Johnson</persName> came to lunch, and we had such a pleasant visit. I went at 5 o'clock over to the <persName>Cleopatra</persName> to tea with the <name>Whittakers</name> &#8211; <persName>Lady Layard,</persName> and the 2 ladies staying with them, <persName>Mr. Johnson</persName> and Capt. <persName>Trefusis</persName> there. Coming home, I overtook <persName ref="#Davis_Theodore_M">Theo</persName> walking, and as the cold, cold wind had much moderated, and he got into the carriage, we drove around the island.</p>
+                <dateline>
+                    <date when="1911-03-16">Thursday. March 16th.</date>
+                </dateline>
+                <p>In town this morning &#8211; met <persName>Artin Pasha,</persName> and Mr. <persName>Sayce</persName> at <placeName>Kyticas's</placeName> &#8211; then went to the <name>Museum</name> &#8211; <persName>M. Daressy</persName> took us into his <pb n="84"/>working room, to show how he was succeeding with the restoring of the beautiful coffin found in Queen <placeName>Tiyi's</placeName> tomb &#8211; a long and very difficult task it is.</p>
```

Comment:
"'s" in "Kyticas's" taken out of tag. Multiple pers refs needed. "the Museum" needs place tag/ref and mention in place index.

Reply:
Do we know what museum she's referring to?

Reply:
It's Boulaq Museum, which is now known as the Cairo Museum. I added this in.